test/models/accessom2.py

```diff
@@ -0,0 +1,92 @@
+"""Specific Access-OM2 Model setup and post-processing"""
+
+import f90nml
```

Comment:
It might be worth leaving a comment saying that `f90nml` is imported implicitly because it's running in the Payu condaenv - just because I'm getting a `Import "f90nml" could not be resolved` warning, but there is no `requirements.txt` file in this repo. Just if someone else comes upon this warning.

Reply:
Left a comment there.